prowler/lib/persistence/manager.py

```diff
@@ -0,0 +1,52 @@
+import os
+
+from .sqlite import SQLiteList, SQLiteDict
+
+default_list = SQLiteList
+
+
+def mklist() -> list:
+    """
+    Create a new list with the given name and data.
+    """
+    prowler_db_connection = os.environ.get('PROWLER_DB_CONNECTION', "memory://")
+    prowler_db_cache_size = os.environ.get('PROWLER_DB_CACHE_SIZE')
+
+    if not prowler_db_connection:
+        return list()
+
+    # In-memory. Default.
+    if prowler_db_connection.startswith('memory://'):
```

Comment:
I don't like using `startswith` here, if the env variable remains as string.

Reply:
Added a TODO